src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,103 @@
+  | `disable_resque`                          | `instrumentation.resque`                  | `instrumentation.resque: disabled`                                                  |
+  | `disable_sinatra`                         | `instrumentation.sinatra`                 | `instrumentation.sinatra: disabled`                                                 |
+  | `disable_rake`                            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_rake_instrumentation`            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_typhoeus`                        | `instrumentation.typhoeus`                | `instrumentation.typhoeus: disabled`                                                |
+
+
+## Removed Support for Ruby 2.2 and 2.3 [#ruby_2223]
+
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above.
+
+
+## Enabled Thread Tracing by Default [#thread_tracing]
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
+If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
+
+
+## Removed Deprecated Instrumentation [#removed_gem_instrumentation]
+
+  Instrumentation for the following gems had been previously deprecated and has now been removed.
+
+  * Acts As Solr
+  * Authlogic
+  * DataMapper
+  * Rainbows
+  * Sunspot
+
+
+## Updated API method `set_transaction_name` [#set_transaction_name]
+
+When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed into the method. This is a change from previous agent versions. 
+  
+Previously, if `set_transaction_name` was called with a new transaction name and a new category that did not match the category already assigned to a transaction, neither the new name nor category would be saved to the transaction. 
+  If this method is being called in a situation in which it was previously ignored due to category differences, this will now change the name and category of the transaction. 
+
+
+## Removed API method [#removed_api_methods]
+
+  The deprecated API method `NewRelic::Agent.disable_transaction_tracing` has been removed. Use `NewRelic::Agent.disable_all_tracing` instead
```

Comment:
The deprecated APIs document references another option, `NewRelic::Agent#ignore_transaction`. Should we list it here too? Is there a reason why someone would want to choose one over the other?

https://github.com/newrelic/docs-website/pull/11408/files#diff-2ce68aa7ce10a595941ab70590bb59697004e62f66c5ac08f22c0af3d6fa2c45R27

Reply:
Nice call out! [Fixed](https://github.com/newrelic/docs-website/pull/11408/commits/0d5f1094361206ade2e96bcc5b0728e03abe8e4e).